charts/karmada/templates/karmada-apiserver.yaml

```diff
@@ -154,6 +154,10 @@ metadata:
   namespace: {{ include "karmada.namespace" . }}
   labels:
   {{- include "karmada.apiserver.labels" . | nindent 4 }}
+  {{- with .Values.apiServer.serviceAnnotations }}
```

Comment:
I am curious why you need an annotation in service. Is there any special use cases?